src/assets/translations/messages-en.json

```diff
@@ -168,7 +170,10 @@
     }
   },
   "staking": {
-    "staking": "Staking"
+    "staking": "Staking",
+    "itWillTake": "It will take",
+    "toUnlock": "to unlock your Osmo after you unstake. Staking rewards will need to withdrawn separately.",
```

Comment:
translations like this should be one translation and template in the string for grammatical and UI formatting reasons

Reply:
How would we handle the span in this case? `XX days` should be bold, so AFAIK there is no way to do that with `react-polyglot`, or at least it's not documented. We would probably need something a la `dangerouslySetInnerHTML`